### PR TITLE
Add start new meeting

### PR DESCRIPTION
**Pull request by mattkrick**
This PR includes more back-end schema refinements.
It also includes the startNewMeeting mutation.

TEST
- [ ] turn on retro release flag, go to team dash, click start retro meeting. Click the green start retro meeting button (it is temporarily enabled even if you shouldn't have access). After clicking it, See the route redirects to /checkin.

### CODE REVIEW

File: src/server/graphql/types/ThinkPhase.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import RetroPhaseItem from 'server/graphql/types/RetroPhaseItem';
+import GenericMeetingStage from 'server/graphql/types/GenericMeetingStage';
+
+const ThinkPhase = new GraphQLObjectType({
+  name: 'ThinkPhase',
+  description: 'The meeting phase where all team members check in one-by-one',
+  interfaces: () => [NewMeetingPhase],
+  fields: () => ({
+    ...newMeetingPhaseFields(),
+    focusedPhaseItemId: {
```
Comment (mattkrick) [inline src/server/graphql/types/ThinkPhase.js]: Following the Meeting -> Phase -> Stage hierarchy, we can store state wherever it makes sense.
for example, where is the facilitator? That's meeting state
which column is highlighted during the think phase? that state only belongs to a single phase
did we already discuss the 3rd card grouping? that state belongs on the stage (isComplete = true)

File: src/universal/components/NewMeetingLobby.js [hunk 2]
```diff
@@ -78,7 +82,7 @@ const NewMeetingLobby = (props: Props) => {
         buttonStyle="solid"
         colorPalette="cool"
         depth={1}
-        disabled={!meetingsRemaining}
+        disabled={canStartMeeting}
```
Comment (mattkrick) [inline src/universal/components/NewMeetingLobby.js]: we'll negate this later, for now, it makes for easy testing

File: src/universal/mutations/StartNewMeetingMutation.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+      meetingId
+      newMeeting {
+        id
+        createdAt
+        facilitatorStageId
+        facilitatorUserId
+        meetingNumber
+        meetingType
+        phases {
+          stages {
```
Comment (mattkrick) [inline src/universal/mutations/StartNewMeetingMutation.js]: See the comment in NewMeetingPhase.js. That allows us to get isComplete here without having to make a new inline fragment to grab it

File: src/universal/mutations/StartNewMeetingMutation.js [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+  const phaseItemSlug = firstPhase.stages.length === 1 ? '' : 1;
+  history.push(`/${meetingTypeSlug}/${teamId}/${phaseSlug}/${phaseItemSlug}`);
+};
+
+const StartNewMeetingMutation = (environment, variables, {history}, onError, onCompleted) => {
+  return commitMutation(environment, {
+    mutation,
+    variables,
+    onError,
+    onCompleted: (res) => {
```
Comment (mattkrick) [inline src/universal/mutations/StartNewMeetingMutation.js]: trying out using the onCompleted & onNext handlers for things that don't need to access the relay store

**Comment by dan-f** [comment]
## Test Cycle 1

- [x] turn on retro release flag, go to team dash, click start retro meeting. Click the green start retro meeting button (it is temporarily enabled even if you shouldn't have access). After clicking it, See the route redirects to /checkin.
  - 💡 Note: I got redirected to `/checkin/1`. Is this correct?

**Comment by dan-f** [comment]
Taking a look at the code next!

**Comment by mattkrick** [comment]
correct! i was thinkin about it, now that the url is a function of the state instead of the state being a function of the url, we could really have some fun with it. maybe stick the name up there in the url instead of a vague index. `/checkin/dan` would be pretty fun.

**Comment by mattkrick** [comment]
merging to master to keep the release train rolling

**Pull request merged**